src/webui/www/private/about.html

```diff
@@ -1,10 +1,677 @@
+<div id="aboutAboutContent" class="aboutTabContent">
+	<img src="images/skin/mascot.png" style="float: right;" alt="qBittorrent Mascot" />
+	<h3>qBittorrent ${VERSION} QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]</h3>
+	<p>QBT_TR(An advanced BitTorrent client programmed in C++, based on Qt toolkit and libtorrent-rasterbar.)QBT_TR[CONTEXT=AboutDialog]</p>
+	<p>Copyright (c) 2011-2018 The qBittorrent project</p>
+	<p>QBT_TR(Home Page:)QBT_TR[CONTEXT=AboutDialog] <a target="_blank" href="https://www.qbittorrent.org"> https://www.qbittorrent.org</a></p>
+	<p>QBT_TR(Bug Tracker:)QBT_TR[CONTEXT=AboutDialog] <a target="_blank" href="http://bugs.qbittorrent.org"> http://bugs.qbittorrent.org</a></p>
+	<p>QBT_TR(Forum:)QBT_TR[CONTEXT=AboutDialog] <a target="_blank" href="http://forum.qbittorrent.org"> http://forum.qbittorrent.org</a></p>
+	<p>QBT_TR(IRC: #qbittorrent on Freenode)QBT_TR[CONTEXT=HttpServer]</p>
+</div>
+
+<div id="aboutAuthorContent" class="aboutTabContent invisible">
+	<h3>QBT_TR(Current maintainer)QBT_TR[CONTEXT=AboutDialog]</h3>
+	<table>
+		<tr>
+			<td>QBT_TR(Name:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td>Sledgehammer999</td>
+		</tr>
+		<tr>
+			<td>QBT_TR(Nationality:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td>QBT_TR(Greece)QBT_TR[CONTEXT=AboutDialog]</td>
+		</tr>
+		<tr>
+			<td>QBT_TR(E-mail:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td><a target="_blank" href="mailto:sledgehammer999@qbittorrent.org">sledgehammer999@qbittorrent.org</a></td>
+		</tr>
+	</table>
+	</br>
+	<h3>QBT_TR(Original authors)QBT_TR[CONTEXT=HttpServer]</h3>
+	<table>
+		<tr>
+			<td>QBT_TR(Name:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td>Ishan Arora</td>
+		</tr>
+		<tr>
+			<td>QBT_TR(E-mail:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td><a target="_blank" href="mailto:ishanarora@gmail.com">ishanarora@gmail.com</a></td>
+		</tr>
+	</table>
+	</br>
+	<table>
+		<tr>
+			<td>QBT_TR(Name:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td>Ankit Gupta</td>
+		</tr>
+		<tr>
+			<td>QBT_TR(E-mail:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td><a target="_blank" href="mailto:ank.iitd@gmail.com">ank.iitd@gmail.com</a></td>
+		</tr>
+	</table>
+	</br>
+	<table>
+		<tr>
+			<td>QBT_TR(Name:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td>Chandan Shikhar Dua</td>
+		</tr>
+		<tr>
+			<td>QBT_TR(E-mail:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td><a target="_blank" href="mailto:shikhar.ap@gmail.com">shikhar.ap@gmail.com</a></td>
+		</tr>
+	</table>
+	</br>
+	<table>
+		<tr>
+			<td>QBT_TR(Name:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td>Swapnil Kumar</td>
+		</tr>
+		<tr>
+			<td>QBT_TR(E-mail:)QBT_TR[CONTEXT=AboutDialog]</td>
+			<td><a target="_blank" href="mailto:swapnil.iitd@gmail.com">swapnil.iitd@gmail.com</a></td>
+		</tr>
+	</table>
+</div>
+
+<div id="aboutSpecialThanksContent" class="aboutTabContent invisible">
+	<p>I would first like to thank sourceforge.net for hosting qBittorrent project and for their support.</p>
```

Comment:
This is all just copy-pasta of `thanks.html`, and same below for `translators.html` and `gpl.html`. Because these html files live in the `gui` folder, I can't load them directly. But if they were in a central location, I could iframe them and avoid so much duplication. Is there somewhere more central we can move them?

Reply:
The fact that WebUI copies GUI is a temporary phenomenon (even though it has been around for a very long time). In fact, they exist in parallel and should not overlap, so this duplication is quite acceptable IMO (it is much more digestible than producing completely unnecessary dependencies).

OFFTOPIC:
Our WebUI is quite archaic. While the use of the Internet is shifting towards mobile devices, we have something designed for wide screens and completely non-scalable.

Reply:
> The fact that WebUI copies GUI is a temporary phenomenon

Not sure I agree- I think closely modeling the Web UI after the GUI is a very deliberate decision (at least on my end).

While the world is definitely trending towards mobile, I think there will continue to be a healthy number of people using desktops/laptops with keyboard, mice, and wide screens. Will this look differently in 5-10 years? Most definitely. But without any desktops + laptops, there's no more qBittorrent at all.

I think there can be a healthy amount of mobile development in the alt webui space. It's just not something I've chosen to spend my time on.

Reply:
>maybe move "src/gui/gpl.html" to "src/license.html"?

We shouldn't put anything in "src" directly according to current structure.
But if something has real base-related meaning in itself then we can put it somewhere in "src/base".

Reply:
I don't feel strongly so will just leave the files as in unless someone objects.